boltdesignsystem/bolt
# Typeahead: Add Footer Customization Option

## Jira
N/A

## Summary
This is the final set of Typeahead updates from https://github.com/boltdesignsystem/bolt/pull/1542 which adds the ability to render custom UI in the footer of the Typeahead component's results.

![image](https://user-images.githubusercontent.com/1617209/68808032-f19fbe00-0636-11ea-9fa8-6c40e1480a0c.png)

## Details
Note: this requires the updates from https://github.com/boltdesignsystem/bolt/pull/1575 and https://github.com/boltdesignsystem/bolt/pull/1567 in order to work as expected. 

The latter is getting pulled in here to show that things are working however would get merged down separately.

## How to test
- Review changes
- Confirm the Typeahead custom footer UI demo is working as expected (including being able to click on the "See All Results" link added as an example)
